app/views/projects/new.html.erb

```diff
@@ -29,10 +29,13 @@ See doc/COPYRIGHT.rdoc for more details.
 <% html_title l("label_project_new") %>
 <%= toolbar title: l(:label_project_new) %>
 <%= labelled_tabular_form_for @project do |f| %>
-  <%= render :partial => 'form', :locals => { :f => f,
-                                              :project => @project,
-                                              :renderTypes => true,
-                                              :show_all_fields => false } %>
+  <%= render partial: 'form', :locals: { f: f,
```

Comment:
`:locals:` ? This breaks current dev specs ;) I've removed this in https://github.com/opf/openproject/commit/0bb88ddb2a5dd33e054ff99dfaba595606aade2b